docs/identity/role-based-access-control/permissions-reference.md

```diff
@@ -1253,7 +1253,7 @@ This is a [privileged role](privileged-roles-permissions.md). Users with this ro
 > | microsoft.directory/serviceAction/activateService | Can perform the "activate service" action for a service |
 > | microsoft.directory/serviceAction/disableDirectoryFeature | Can perform the "disable directory feature" service action |
 > | microsoft.directory/serviceAction/enableDirectoryFeature | Can perform the "enable directory feature" service action |
-> | microsoft.directory/serviceAction/getAvailableExtentionProperties | Can perform the getAvailableExtentionProperties service action |
```

Comment:
Please revert this change

Reply:
Reverted. Is this something you can bring back up the tree so they fix it ?
As the source is spelled correctly - https://learn.microsoft.com/en-us/graph/api/directoryobject-getavailableextensionproperties?view=graph-rest-1.0&tabs=http